### ISSUE TITLE
Crashed upon first run directly after installing

### ISSUE THREAD

**Issue by GigglePocket** [sumatrapdfreader/sumatrapdf#3003]
[Crash Log Gist](https://gist.github.com/GigglePocket/36ec185da6176c1e8fcf6ff15fd91c28)

> include the file that caused the crash
- Run SumatraPDF was selected from the installation menu so no file was selected for opening

> what were you doing when the crash happened
- Drinking coffee
- In all seriousness though, the only other application running on my computer during installation/running SumatraPDF was Chrome (I know I know, I'm switching to Firefox soon, I promise)

> when did the crash happen. When opening a file? changing view? etc.
- Directly after the installation upon selecting the "Run SumatraPDF" option from the installation menu

> how did you open the file? drag & drop on Sumatra window? Double-click in file manager? From command line?
- See above

> the best information is a set of steps I can do to reproduce the crash
1) Download SumatraPDF from [this](https://www.sumatrapdfreader.org/dl/prerel/15249/SumatraPDF-prerel-64-install.exe) link
2) Run the executable
3) Install with all options checked
4) Select "Run SumatraPDF"

### Comments
- All subsequent attempts to run SumatraPDF yielded the same result
- Thank you for developing/maintaining this software. I appreciate what you do.

**Comment by GitHubRulesOK** [comment]
While drinking next java cup look at any one of the many similar issues 
(yes it could be posted on the download page but without access I don't build it just support by comment)
A good place to start is https://github.com/sumatrapdfreader/sumatrapdf/issues/2984

**Comment by GigglePocket** [comment]
Thank you! I was getting sidetracked 7 ways from Sunday from the project I'm trying to complete when I found the perfect book documenting the system/framework I'm using, and upon the umteenth pdf viewer failing, I decided to quickly look for, install, and use a decent reader for PDF/epub files, so when I ran into the crash and the link to submit an issue I got lazy and just went through the motions without searching for my issue 'cause I just wanted to get back to my intended task. 

I downloaded the latest stable release of SumatraPDF and it works perfectly! 

Thank you again and, apologies for my lazyness.

I'm closing this issue as, like you said, there are already many similar issues and this would only distract from one that might be fruitful.
